components/rhel/rhel-ose/Vagrantfile

```diff
@@ -86,8 +86,10 @@ Vagrant.configure(2) do |config|
     echo "You can now access the OpenShift console on: https://${OSIP}:8443/console"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "$ oc login ${OS_IP}:8443"
+    echo
+    echo "To download and install OC binary, run:"
+    echo "vagrant service-manager install-cli openshift"
```

Comment:
is `To download and install OC binary, run:` or should we make it as `If you don't have OpenShift CLI, OC, install it as` 

@Preeticp @thatdocslady comments?